app/controllers/org_admin/templates_controller.rb

```diff
@@ -376,7 +376,7 @@ def template_options()
           # Load the funder's template(s)
           templates = Template.valid.publicly_visible.where(published: true, org_id: funder_id).to_a
 
-          if org_id.present?
+          unless org_id.blank?
```

Comment:
if funder_is is not blank (L375) there is not need to check if org_id is not blank (L379) since you will never enter in the if (L374).

Reply:
I don't see the point of having the if statement (L374) when you have two conditional within it to check funder_id.present? and after org_id.present?.
- present? is the negation of blank?
- blank can be empty string, empty string with spaces, [], {}, false and nil.

Anyway, it works but just to note it.